ui/app/(prowler)/providers/(set-up-provider)/update-credentials/page.tsx

```diff
@@ -17,31 +17,51 @@ interface Props {
   };
 }
 
+// Helper function to determine if the credentials form should be shown
+const shouldShowCredentialsForm = (
```

Comment:
same hare, we don't want helper functions in a page

Reply:
Can you answer this @alejandrobailo, please? I think you added that part.